packages/react/src/components/ContextualMenu/ContextualMenuItemWrapper/ContextualMenuSplitButton.tsx

```diff
@@ -167,6 +167,7 @@ export class ContextualMenuSplitButton extends ContextualMenuItemWrapper {
     const { contextualMenuItemAs: ChildrenRenderer = ContextualMenuItem, onItemClick } = this.props;
 
     const itemProps: IContextualMenuItem = {
+      ...item,
```

Comment:
it looks like itemProps (as itemComponentProps) is getting pulled out of item and passed into the childRender directly, whereas this new itemProps is being passed into the 'item' of the child renderer. We should probably remove the itemProps from item, i'm curious if they would cause any other bugs.